nixos/modules/services/misc/autobrr.nix

```diff
@@ -0,0 +1,106 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.autobrr;
+in
+{
+  options = {
+    services.autobrr = {
+      enable = mkEnableOption (lib.mdDoc "Autobrr");
+
+      dataDir = mkOption {
+        type = types.path;
+        default = "/var/lib/autobrr";
+        description = lib.mdDoc ''
+            The directory where Autobrr will create files.
+          '';
+      };
```

Comment:
This is not used outside of the `autobrr` user home, and being created by tmpfiles.